application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -444,6 +444,18 @@ type Emissions {
   co2: Grams
 }
 
+"Station entrance or exit."
+type Entrance {
+  "Short text or a number that identifies the entrance or exit for passengers. For example, `A` or `B`."
+  code: String
+  "ID of the entrance in the format of `FeedId:EntranceId`."
```

Comment:
Same here, you could mention that if the feed is `osm`, the entrance is from OSM data.